libs/wingc/src/jsify/snapshots/base_class_with_fields_preflight.snap

```diff
@@ -64,6 +66,7 @@ const $platforms = ((s) => !s ? [] : s.split(';'))(process.env.WING_PLATFORMS);
 const $outdir = process.env.WING_SYNTH_DIR ?? ".";
 const $wing_is_test = process.env.WING_IS_TEST === "true";
 const std = $stdlib.std;
+const $helpers = $stdlib.helpers;
```

Comment:
I am wondering if perhaps it would make more sense to put the helpers in `std`

Reply:
Totally, yeah I think that's the next step. Eventually we can have a place named std which just has shared utilities and types (like Datetime) that should be included in every inflight code bundle. Anything that doesn't need to be bundled like std.Test / TestRunner can be moved elsewhere perhaps